src/System.EventLog/src/System/Diagnostics/UnsafeNativeMethods.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.Win32.SafeHandles;
+
+namespace Microsoft.Win32
+{
+    internal static class UnsafeNativeMethods
+    {
+        [DllImport(Interop.Libraries.Advapi32, CharSet = CharSet.Unicode, SetLastError = true)]
+        public static extern bool ReportEvent(SafeHandle hEventLog, short type, ushort category,
+                                                uint eventID, byte[] userSID, short numStrings, int dataLen, HandleRef strings,
+                                                byte[] rawData);
+#pragma warning disable BCL0015 // Disable Pinvoke analyzer errors. 
+        [DllImport(Interop.Libraries.Advapi32, CharSet = CharSet.Unicode, SetLastError = true)]
+        public static extern bool ClearEventLog(SafeHandle hEventLog, HandleRef lpctstrBackupFileName);
+        [DllImport(Interop.Libraries.Advapi32, CharSet = CharSet.Unicode, SetLastError = true)]
```

Comment:
Please put a blank line between each pair of lines so it's mor readable

Reply:
Done